index.html

```diff
@@ -1,142 +1,125 @@
-      </div>
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-12 u-sv3">
+      <h2 class="p-muted-heading">Features</h2>
     </div>
-  </section>
-
-  <section class="p-strip--light">
-    <div class="row">
-      <div class="col-12 u-sv3">
-        <h2 class="p-muted-heading">Features</h2>
+  </div>
+  <div class="row">
+    <div class="col-4">
+      <div class="u-sv1">
+        <img src="https://assets.ubuntu.com/v1/aeec264b-green-tick.svg" alt="" />
       </div>
+      <h3 class="p-heading--four">Develop for Ubuntu</h3>
+      <p>Ubuntu is the leading operating system in the cloud and IoT. With Multipass you can target it directly from your developer OS of choice.</p>
     </div>
-    <div class="row">
-      <div class="col-4">
-        <div class="u-sv1">
-          <img src="https://assets.ubuntu.com/v1/aeec264b-green-tick.svg" alt="" />
-        </div>
-        <h3 class="p-heading--four">Develop for Ubuntu</h3>
-        <p>Ubuntu is the leading operating system in the cloud and IoT. With Multipass you can target it directly from your developer OS of choice.</p>
+    <div class="col-4">
+      <div class="u-sv1">
+        <img src="https://assets.ubuntu.com/v1/aeec264b-green-tick.svg" alt="" />
       </div>
-      <div class="col-4">
-        <div class="u-sv1">
-          <img src="https://assets.ubuntu.com/v1/aeec264b-green-tick.svg" alt="" />
-        </div>
-        <h3 class="p-heading--four">Always up to date</h3>
-        <p>Multipass always uses the newest and most up-to-date image available, and will maintain those you use often.</p>
-      </div>
-      <div class="col-4">
-        <div class="u-sv1">
-          <img src="https://assets.ubuntu.com/v1/aeec264b-green-tick.svg" alt="" />
-        </div>
-        <h3 class="p-heading--four">Easy to use</h3>
-        <p>Getting a new instance is as easy as <code>multipass launch</code>.</p>
+      <h3 class="p-heading--four">Always up to date</h3>
+      <p>Multipass always uses the newest and most up-to-date image available, and will maintain those you use often.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-sv1">
+        <img src="https://assets.ubuntu.com/v1/aeec264b-green-tick.svg" alt="" />
       </div>
+      <h3 class="p-heading--four">Easy to use</h3>
+      <p>Getting a new instance is as easy as <code>multipass launch</code>.</p>
     </div>
-  </section>
+  </div>
+</section>
 
-  <section class="p-strip">
-    <div class="row">
-      <div class="col-12">
-        <h2 style="font-weight: 100;">Get started</h2>
-        <p>Multipass is available for <a href="/install#macOS">macOS</a> and <a href="/install#windows">Windows</a>, and the many <a href="/install#snap">Linux distributions</a> that support <a href="https://snapcraft.io/" class="p-link--external">snaps</a>.</p>
-      </div>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2 style="font-weight: 100;">Get started</h2>
```

Comment:
why put this inline?

Reply:
There are q few more of these in this and other files

Reply:
This is a quick hack used in other places until Vanilla 2.0 is released.